CMakeLists.txt

```diff
@@ -178,6 +178,15 @@ set_target_properties(fmt PROPERTIES
   VERSION ${FMT_VERSION} SOVERSION ${CPACK_PACKAGE_VERSION_MAJOR}
   DEBUG_POSTFIX d)
 
+# Configure pkg-config fmt.pc properly
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
```

Comment:
Please use `fmt` instead of `PROJECT_NAME_LOWER` because the project name is not related to the library target or output name (although they happen to be the same).